portal/portal_payment.php

```diff
@@ -453,33 +538,30 @@ function notifyPatient() {
     ?>
     <title><?php echo xlt('Record Payment'); ?></title>
     <style type="text/css">
-        body {
-            /* font-family:sans-serif; font-size:10pt; font-weight:normal */
-        }
-
         .dehead {
-            color: #000000; /*font-family:sans-serif; font-size:10pt;*/
+            color: #000000;
             font-weight: bold
         }
-
         .detail {
             padding: 1px 1px;
-            /* width: 65px; */
-            color: #000000; /*font-family:sans-serif; font-size:10pt; */
+            color: #000000;
             font-weight: normal
         }
     </style>
-    <script type="text/javascript"
-            src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-creditcardvalidator/jquery.creditCardValidator.js"></script>
-    <script type="text/javascript"
-            src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
-
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-creditcardvalidator/jquery.creditCardValidator.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
     <script type="text/javascript">
-        var mypcc = '1';
+        var chargeMsg = <?php $amsg = xl('Payment was successfully authorized and your card is charged.') . "\n" .
+                xl("You will be notified when your payment is applied for this invoice.") . "\n" .
+                xl('Until then you will continue to see payment details here.') . "\n" . xl('Thank You.');
+            echo json_encode($amsg); // backward compatable 5.0.1
```

Comment:
I'm pretty sure this will break the newlines. If so, let me know, and will show how can get this escaped.

Reply:
I'd be surprised. remember json_encode is a super set of the spec for PHP but i'll check on demo.